diem-unified-charter.md

```diff
@@ -12,47 +9,44 @@ ISO 7010 defines a set of emblems that can be used to identify hazards, these in
 The presence of a digital emblem represents a new signal available to cyber operators. 
 
-The DIEM WG will define an architecture, set of emblem types, and presentation methods for creating, displaying, validating and interpreting emblems across applications and platforms in a cohesive way.
-The initial scope of the working group will assume that an emblem is also discovered through digital communication, however, ideally the methods defined for emblem creation and validation should not depend on this assumption.
+The DIEM WG will define an architecture and discovery mechanism enabling digital emblems to be presented and validated across applications and platforms in a cohesive way.
 
 # Architectural Consideration
 
-"To bear an emblem" means to present or display a symbol, figure, or device as an identifying mark. 
+"To issue or produce" an emblem, is to design the identifying mark and its associated attributes.
+The emblem issuer's role is to ensure that the identifying mark is easy to recognize and well understood.
+
+"To bear an emblem" means to present or display an identifying mark. 
 The entity that bears the emblem is respectively the bearer or emblem holder. 
 This can be a separate entity from the emblem issuer.
 
 "To validate an emblem" means to confirm the authenticity or legitimacy of a particular symbol or design, often by checking its details against a known standard or reference point. 
-Emblems may be observed by validators without the knowledge of the bearer displaying the emblem, or may be presented to a specific verifier upon request.
-Cryptographic verification can be optional as long is the emblem can be correctly interpreted.
+Emblems may be observed by validators without the knowledge of the bearer displaying the emblem, or may be presented to a specific validator upon request.
+Cryptographic verification can be optional as long as the emblem can be correctly interpreted.
 Therefore, to be effective, the semantics of an emblem must be well known, easily recognizable, and distinguishable from other emblems.
 
-Digital emblems can be bound to entities that are analog (e.g., a vehicle) or digital (e.g., a web page), however, there needs to be a digital representation that is used for validation using digital communication.
-
-Digital emblems can fall into multiple types (e.g., self-signed, attested, etc.) that will affect how a validator interprets and trusts the assertions made within the emblem.
-The communication between an entity that bears an emblem and a validator will influence requirements on how validators retrieve digital emblems. 
-These requirements will be key drivers in the development of a digital emblem architecture.
+Digital emblems can be borne or presented by entities that are analog (e.g., a vehicle) or digital (e.g., a web page), however, there needs to be a digital representation that is used for validation using digital communication.
```

Comment:
I reworked this... Its still vague in the sense of "are the people in the vehicle marked" or "are adjacent services published in the web page marked", but thats too much detail for a charter probably.

Reply:
Why introduce a new verb ("borne") here?

However, as I also said previously, I'd be okay to remove this sentence.